src/connectors/argentMobile/index.ts

```diff
@@ -56,18 +56,19 @@ export class ArgentMobileBaseConnector extends Connector {
   }
 
   async ready(): Promise<boolean> {
-    // check if session is valid and retrieve the wallet
-    // if no sessions, it will show the login modal
-    await this.ensureWallet()
```

Comment:
Without this code the `_wallet` property will not be initialized unless `connect` is called - will this be good enough? I wonder if the `ensureWallet` method shouldn't still initialize the `_wallet` property but not try to call `enable`

Reply:
> Without this code the _wallet property will not be initialized unless connect is called - will this be good enough?

yes, it follows the same behaviour as webwallet

- if the wallet is not initialized, the dapp would display a `connect` button 
- if the dapp has an autoconnect function, it will be initialized 

if the dapp perform the request while not connected, it will receive the exception that will be managed on dapp side (like showing the connect modal)

managing `ensureWallet` without enable could be a little tricky at the moment, but can be a good point to improve

Reply:
Sounds good to me - just wanted to make sure it won't break our normal flows 👍